guides/source/plugins.md

```diff
@@ -390,7 +390,7 @@ class ActsAsYaffleTest < ActiveSupport::TestCase
 end
 ```
 
-Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method `squawk'",
+Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method 'squawk'",
```

Comment:
Opening backticks can be escaped with `\`: 

```suggestion
Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method \`squawk'",
```

Reply:
Huh, I didn't realize that the Ruby output was actually a backtick 😯

Reply:
Indeed, though there is some talk of changing it: https://bugs.ruby-lang.org/issues/16495